models/nft/ethereum/nft_ethereum_aggregators_markers.sql

```diff
@@ -0,0 +1,96 @@
+{{ config(
+        alias ='aggregators_markers',
+		materialized = 'view',
+        unique_key='hash_marker',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["hildobby"]\') }}')
+}}
+
+ WITH reservoir AS (
+    SELECT distinct substring(unhex(regexp_replace(data, '^.*00', '')), 2, length(unhex(regexp_replace(data, '^.*00', '')))-2) AS router_website
+    , regexp_replace(data, '^.*00', '') AS hash_marker
+    FROM {{ source('ethereum','transactions') }}
+    WHERE to IN (
+        '0x00000000006c3852cbef3e08e8df289169ede581', --seaport
+        '0x74312363e45dcaba76c59ec49a7aa8a65a67eed3', --x2y2
+        '0x59728544b08ab483533076417fbbb2fd0b17ce3a', --looksrare
+        '0x9ebfb53fa8526906738856848a27cb11b0285c3f'  --reservoir
+    )
+    AND RIGHT(data, 2) = '1f'
+    AND LEFT(regexp_replace(data, '^.*00', ''), 2)='1f'
+    AND regexp_replace(data, '^.*00', '') != '1f'
+    AND length(regexp_replace(data, '^.*00', ''))%2 = 0
+    AND block_time > '2022-10-15'
+    )
+
+SELECT hash_marker
+, 'Reservoir' AS aggregator_name
+, CASE WHEN router_website='ens.vision' THEN 'ENS.Vision'
```

Comment:
fwiw, this could have been a nice place to create a separate static spell with two cols (router website and name) that you join onto here. that makes the effort put into this list more modular. but I'm not asking you to wait another 2.5 hours to test that, just a comment.